python_on_whales/components/compose/cli_wrapper.py

```diff
@@ -1005,6 +1009,8 @@ def up(
         full_cmd.add_flag("--build", build)
         full_cmd.add_flag("--detach", detach)
         full_cmd.add_flag("--wait", wait)
+        if wait_timeout:
+            full_cmd.add_simple_arg("--wait-timeout", wait_timeout)
```

Comment:
```suggestion
        full_cmd.add_simple_arg("--wait-timeout", wait_timeout)
```
`àdd_simple_arg` doesn't do anything if the second argument is None